ush/create_ufs_configure_file.py

```diff
@@ -62,35 +96,42 @@ def create_ufs_configure_file(run_dir):
     #-----------------------------------------------------------------------
     #
     settings = {
-      "dt_atmos": DT_ATMOS,
-      "print_esmf": PRINT_ESMF,
-      "cpl_aqm": CPL_AQM,
-      "pe_member01_m1": pe_member01_m1,
-      "aqm_pe_member01_m1": aqm_pe_member01_m1,
-      "atm_omp_num_threads": OMP_NUM_THREADS_RUN_FCST,
+      "ufs_fire": cfg["UFS_FIRE"],
+      "logKindFlag": logkindflag,
+      "EARTH_cl": earth_component_list,
+      "ATM_pb": atm_petlist_bounds,
+      "ATM_omp_num_threads_line": atm_omp_num_threads_line,
+      "ATM_diag_line": atm_diag_line,
+      "runseq": runseq,
+      "AQM_pb": "",
+      "FIRE_pb": "",
+      "dt_atmos": cfg["DT_ATMOS"],
+      "print_esmf": cfg["PRINT_ESMF"],
+      "cpl_aqm": cfg["CPL_AQM"]
     }
-    settings_str = cfg_to_yaml_str(settings)
+    if cfg["CPL_AQM"]:
+        settings["AQM_pb"] = aqm_petlist_bounds
+    if cfg["UFS_FIRE"]:
+        settings["FIRE_pb"] = fire_petlist_bounds
 
     print_info_msg(
         dedent(
             f"""
-            The variable \"settings\" specifying values to be used in the \"{UFS_CONFIG_FN}\"
+            The variable \"settings\" specifying values to be used in the \"{cfg["UFS_CONFIG_FN"]}\"
             file has been set as follows:\n
-            settings =\n\n"""
-        )
-        + settings_str,
-        verbose=VERBOSE,
+            {settings=}\n\n"""
+        ), verbose=cfg["VERBOSE"]
     )
     #
     #-----------------------------------------------------------------------
     #
-    # Call a python script to generate the experiment's actual UFS_CONFIG_FN
-    # file from the template file.
+    # Call uwtools "render" set_template function from workflow_tools to fill in jinja template
+    # from config template and settings variable to create ufs.configure file for this experiment
```

Comment:
```suggestion
    # Call the uwtools "render" function to fill in jinja expressions contained in the UFS Configure file template
    # with the values from the settings variable to create ufs.configure file for this experiment
```

Would you mind using the term `jinja expressions` here? It would make it more consistent with uwtools documentation and may help users who are cross-referencing docs. In general, we've been trying to refer to the `{{ }}` entries as Jinja expressions, and then the file that contains them would be a template file.